react-todo/src/components/utils/checkLocalStorage.ts

```diff
@@ -0,0 +1,22 @@
+export const checkLocalStorage = () => {
+  const today = new Date();
+  const stringToCompare = `${today.getDate()}-${today.getMonth()}`;
+  console.log(stringToCompare);
+  let lastDateModalShown;
+  if (localStorage.lastDateShown) {
```

Comment:
please add empty lines to the code, it's hard to read when there are no logic blocks